style.css

```diff
@@ -0,0 +1,415 @@
+  margin-left: 29px;
+  margin-top: 22px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 30px;
+  line-height: 37px;
+  text-align: center;
+  letter-spacing: 0.07em;
+  color: #fff;
+}
+
+.header__menu {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -ms-flex-item-align: end;
+  align-self: flex-end;
+  width: 70px;
+  height: 70px;
+  background: #f5f5f3;
+  cursor: pointer;
+}
+
+.header__burger {
+  -ms-flex-item-align: center;
+  -ms-grid-row-align: center;
+  align-self: center;
+  margin: auto;
+}
+
+.header__burger-line {
+  display: block;
+  width: 26px;
+  height: 3px;
+  background: #000000;
+}
+
+.header__burger-line:first-of-type {
+  margin-bottom: 5px;
+}
+
+button {
+  background: none;
+}
+
+.up-section__btn_else,
+.up-section__btn_offer {
+  text-transform: uppercase;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+}
+
+.up-section__btn.up-section__btn_else {
+  width: 203px;
+  height: 30px;
+  padding-top: 14px;
+  border: 1.5px solid #000000;
+  justify-content: center;
+}
+
+.up-section__btn_else:hover {
+  background-color: #eeddd0;
+}
+
+.up-section {
+  position: relative;
+  width: 100%;
+}
+
+.up-section__title {
+  position: relative;
+  z-index: 10;
+  padding-left: 26px;
+  margin-top: -165px;
+  font-weight: bold;
+  line-height: 51px;
+  font-size: 50px;
+}
+
+.up-section__description,
+.up-section__contacts {
+  display: none;
+}
+
+.up-section__images {
+  z-index: -10;
+  height: 529px;
+  overflow: hidden;
+}
+
+.up-section__collection-image img {
+  width: 100%;
+  height: 100%;
+      -o-object-fit: cover;
+  object-fit: cover;
+      -o-object-position: 0 -55px;
+  object-position: 0 -55px;
+}
+
+.up-section__desription {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 9px;
+  line-height: 13px;
+  color: #000000;
+}
+
+.up-section__scrooll {
+  position: absolute;
+  left: 0;
+  right: 0;
+  bottom: -40px;
+  width: 28px;
+  height: 18px;
+  margin: auto;
+  background-repeat: no-repeat;
+  background-position: center;
+  cursor: pointer;
+}
+
+@media (min-width: 992px) {
+  .header {
+    min-height: 92px;
+  }
+
+  .header__head {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+      -webkit-box-pack: justify;
+      -ms-flex-pack: justify;
+    justify-content: space-between;
+    min-width: 461px;
+    background: #000000;
+  }
+
+  .header__logo {
+    margin-top: 42px;
+    margin-left: 55px;
+  }
+
+  .header__lang {
+    position: relative;
+    display: block;
+    width: 50px;
+      -ms-flex-item-align: center;
+      -ms-grid-row-align: center;
+    align-self: center;
+    margin-right: 22px;
+    margin-top: 20px;
+  }
+
+  .header__lang-option {
+    position: absolute;
+    width: 31px;
+    height: 25px;
+    left: 388px;
+    top:0;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    text-align: right;
+    text-decoration: none;
+    color: #FFFFFF;
```

Comment:
Codacy found an issue: [Expected "#FFFFFF" to be "#FFF" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/locoalin/pullRequest?prid=5676737)